upstash/README.md

```diff
@@ -0,0 +1,38 @@
+# Upstash
+
+## Overview
+
+Upstash is a serverless data provider enabling Redis®, Kafka, and messaging/scheduling solutions for a diverse range of applications that provides speed, simplicity, and a seamless developer experience. 
+
+Upstash utilizes the Redis and Kafka APIs, and is designed for:
+
+* Serverless functions (AWS Lambda)
+* Cloudflare Workers
+* Fastly Compute@Edge
+* Next.js Edge, Remix, and more
+* Client-side web or mobile applications
+* AI development
+* WebAssembly and other environments where HTTP is preferred over TCP connections
+
+## Setup
+
+### Installation
+
+Visit [Upstash][4] to sign up for free. Once registered, visit the [Upstash integration tile][5] in Datadog and install the integration. Once installed, navigate to the **Configure** tab and click **Connect Accounts**. This will guide you through the Datadog OAuth flow to grant Upstash access to your database metrics.
+
+### Uninstallation
+
+To remove the Datadog integration from Upstash, navigate to the [Upstash integrations page][1] and click **Remove**. Additionally, uninstall this integration from Datadog by clicking the **Uninstall Integration** button on the [integration tile][5]. Once you uninstall this integration, any previous authorizations are revoked.
+
+Additionally, ensure that all API keys associated with this integration have been disabled by searching for `upstash` on the [API Keys management page][3].
+
+
+Need help? Contact [Upstash support][7].
```

Comment:
```suggestion
## Data Collected

### Metrics

See [metadata.csv][6] for a list of metrics provided by this integration.
### Events
The Upstash integration does not include any events.
 
### Service Checks

The Upstash integration does not include any service checks.

## Support
Need help? Contact [Upstash support][7].
```

Reply:
Just a friendly bump on these changes

Reply:
@WoodProgrammer can you please commit these changes as well? thanks!